File: lib-segment/src/main/java/websiteschema/mpsegment/dict/POSArrayFactory.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package websiteschema.mpsegment.dict;

/**
 *
 * @author twer
 */
public class POSArrayFactory {

    private static final POSArray posArray = new POSArray();

    static {
        POS pos = new POS(POSUtil.getPOSString(POSUtil.POS_UNKOWN), 1);
        posArray.add(pos);
    }

    static POSArray getUnknownWordPOSArray() {
        return posArray;
    }
}

File: lib-segment-scala/src/main/scala/websiteschema/mpsegment/dict/POSArrayFactory.scala
package websiteschema.mpsegment.dict

object POSArrayFactory {

  private val posArray = new POSArray()

  var pos = POS(POSUtil.getPOSString(POSUtil.POS_UNKOWN), 1)
  posArray.add(pos)

  def getUnknownWordPOSArray() = posArray

}
